EnglishFilter/sections/foreign.txt

```diff
@@ -3782,6 +3782,8 @@ bhaskar.com#%#var canABP = true;
 !
 ! https://github.com/AdguardTeam/AdguardFilters/issues/74675
 nawalakarsa.id#?#.jeg_header > .jeg_bottombar > .container > .jeg_nav_row > .jeg_nav_col > .item_wrap > .jeg_ad:upward(.jeg_bottombar)
+! https://github.com/AdguardTeam/AdguardFilters/issues/74930
+github.io#%#//scriptlet("prevent-setInterval", "chkAds", "2500")
```

Comment:
cannot be used on  iOS
> github.io#@#iframe[id^="google_ads_iframe"]

too wide exclusion, and redundant, as I see.
Added a bit different rules.

Reply:
Ok.